subiquity/curtin.py

```diff
@@ -25,7 +25,7 @@
 
 TMPDIR = '/tmp'
 CURTIN_SEARCH_PATH = ['/usr/local/curtin/bin', '/usr/bin']
-CURTIN_INSTALL_PATH = ['/media/root-ro', '/']
+CURTIN_INSTALL_PATH = ['/media/root-ro', '/rofs', '/']
```

Comment:
Furthemore, /rofs is *not* the CD but a pristine live system, exactly as it should be once installed to disk. Another squashfs layer exists for the installer bits, and any changes done while in the installer environment as simply lost (we don't keep the overlay-work directories, I in fact have no idea where they are -- we don't need them anyway... we can revisit this if we want persistence in the future).